### ISSUE TITLE
[Suggestion] YTTR compatibility with BetterNether: Special Biome generation aproach

### ISSUE THREAD

**Issue by Shibva** [quiqueck/BetterNether#158]
### Description

It appears that part of BetterNether's generation caused some of YTTR's generation to be overridden to a large extent.

At first, I just thought the mod was just causing the bedrock roof to generate; but on discovered on further investigation that it increases the max height of the Nether's roof; causing a generation overlap with YTTR's Scorched and Scorched Sumit which occurs slightly at and above y=128 (original roof height)

this may not be much of an issue but it has been claimed that this generation would be vital for future progression to be implemented soon.

### What is the Scorched and Scorched Sumit and what does it have todo with BetterNether?
These two biomes are added by YTTR as part of its gimmick for The Nether; these biomes and custom generation remove the Nether's Roof and generate these biomes on top of it, The Scorched being the lower section of the height above the roof and the Scorched Sumit being the upper part of it up to the build limit. Its said that mobs won't spawn up there because of the scorching heat present and some structure generation also occurs here. 

After reading this you will realize the issue since BetterNether has a higher Nether.

### What do you want us to do about it?
I may not know much about coding mods and stuff, but Im going to speculate that its safe to say that fixing this bug is not an option; so why not make it a feature?

Firstly have it to where the Bedrock Ceiling generation is disabled if yttr is present and its config value for enabling this generation is true 

As for what you could do, you could create a custom biome that can make up for the loss of the Scorched

Scorched Plateaus: The alternate for implementing the Scorched biome added by YTTR. These areas tend to have lots of plateau features consisting of YTTR's ideal Scorched Biome generation. This area could have a lower ceiling to allow for the Scorched Sumit to exist above it.

Scorched Sumit: can consist above BetterNethers roof of the Nether and can be found anywhere above BetterNether's Bedrock Ceiling, it's not much space but still enough as there is still some generation that occurs up there; there will be more room in biomes with the Scorched Plateaus 

### If there anything else I need to know?
That about it afaik; though I will also make a "sister report" on YTTR's end regarding this issue and reference this page once published.

I hope that something can be worked out as both mods are great

**Comment by Shibva** [comment]
TLDR: 

Yttr extends the nether orginaly by adding stuff that was originally above the nether roof; but it cant with that being changed

Some biome integration would be nice if that's possible
